### ISSUE TITLE
push an 1.3.0 based build to the herd

### ISSUE THREAD

**Issue by simonthum** [renatoathaydes/parcey#10]
There's this issue that have to go `native("jvm")` in oder to use the current 0.4.0 due to what I'd like to call js-backend ABI differences if it wasn't text.

I assume rebuilding 0.4.0. with the current compiler would fix the cross-platform use case:

Version 0.4.0 of module com.athaydes.parcey was compiled by an incompatible version of the compiler (binary version 8. of module is not compatible with binary version 9. of this compiler)

native(jvm) fixes it for me, but that isn't what I have in mind.

**Comment by renatoathaydes** [comment]
yeah, Parcey is getting old :) will publish a new version soon... feel free to check out the code and build it locally for now!

**Comment by simonthum** [comment]
Cool, thanks for keeping up with it! I'll have a bit of screen-free time now, so if you update before next week I'll be fine. But I'm sure it's easy to build just in case.
